src/app/models/history.model.ts

```diff
@@ -0,0 +1,39 @@
+export interface Category {
+  id: string;
+  name: string;
+  capacity: number;
+  userId: number;
+}
+
+export type TransactionType = 'income' | 'outcome';
+
+export interface Event {
+  id: string;
+  type: TransactionType;
+  amount: number;
+  category: number;
+  date: string;
+  description: string;
+  userId: number;
```

Comment:
in your db.json I didn't find any userId, so when you load events how do you know that they relate to a specific user?